### DISCUSSION TITLE
Using/Downloading/Displaying Videos

### DISCUSSION THREAD

**Discussion by guyas** [SafeExamBrowser/seb-win-refactoring#800] [Q&A] [answered]
After successfully logging into our system, there's a screen with a short and lightweight (377K) splash video, which comes in MP4 format.
When opening the system in a "regular" chrome, it quickly loads the video, plays it then progresses to the next part.
![image](https://github.com/SafeExamBrowser/seb-win-refactoring/assets/8017654/53c5dc60-2f87-4972-991f-ab6c67a83c5c)

First request loads bytes 0-385513/385514 (length 385514)
Second 360448-385513/385514 (length 25066)
Third 32768-385513/385514 (length 352746)

When accessing the very same place on the very same computer with the SEB, the splash video doesn't show and it all gets stuck. Sometimes it gets the file with the same sequence of 206 statuses, sometimes it cancels the download:
![Capture2](https://github.com/SafeExamBrowser/seb-win-refactoring/assets/8017654/99009b0a-5d6e-4e00-9fe1-4c62d566af3a)
At both scenarios, the video does not show. I tried enabling downloads, disabling cache deletion...   nothing helped.

Then I suspected something was broken, so I looked for a website that also uses videos, such as [this WaPo item](https://www.washingtonpost.com/dc-md-va/interactive/2024/loudoun-county-gravel-roads-national-register-historic-places/?itid=hp-top-table-main_p001_f006). It looks like this (video background):
![image](https://github.com/SafeExamBrowser/seb-win-refactoring/assets/8017654/154f536d-15ce-4606-bd0b-7cb7e8e6d27d)

Then I configured the SEB to go to that url, and this is the screen I got:
![wapo](https://github.com/SafeExamBrowser/seb-win-refactoring/assets/8017654/3d90d48b-72cb-463d-a0d0-0fce342dc319)
and this is the networking performance:
![wapoerr](https://github.com/SafeExamBrowser/seb-win-refactoring/assets/8017654/3c273c7a-d706-4099-964a-27b4a8927c76)

At this point I'm afraid there's something terribly wrong with my configuration of how to display media, but I don't know what.

**Comment by danschlet** [comment]
SEB for Windows cannot display mp4, use WebM. Search Issues for the reason why.

**Reply by guyas** [reply]
Is this a bug? A feature?

Also, what about other media types (e.g. OGG)?

**Reply by danschlet** [reply, marked as answer]
https://github.com/search?q=repo%3ASafeExamBrowser%2Fseb-win-refactoring+mp4&type=issues
https://github.com/SafeExamBrowser/seb-win-refactoring/discussions?discussions_q=mp4

**Reply by guyas** [reply]
OK this covers the point, thanks. We'll see what is our ability to convert stuff into open formats. Thanks!
